### PR TITLE
fixes parsing of flf files when last line is empty

### LINKED ISSUE #5: pyramid and maxfour font is broken ?
$ artii -l | grep max
    maxfour

    $ artii -f maxfour sample
    Something has gone horribly wrong!
    Artii says: undefined method `empty?' for nil:NilClass

    $ artii -l | grep pyramid
    pyramid

    $ artii -f pyramid sample
    Something has gone horribly wrong!
    Artii says: undefined method `empty?' for nil:NilClass

Was able to correctly display fonts other than this

### PR DESCRIPTION
**Pull request by common-nighthawk**
i believe this fixes #5.  it should now be possible to bring back fuzzy and wavy!  (some other live fonts were also broken--like maxfour.)

#load_extended_characters iterates over the last lines of the file, and tries to parse it as long as the line is not blank.

however, `if i && i.empty?` will not provide the expected result if the line is `""` (in the file) aka `nil` (in ruby).  rails has #blank, which would be perfect.  but `!i || i.empty?` should work.
http://apidock.com/rails/Object/blank%3F

in short--it was the flf files with totally blank lines at the end that were breaking.

### CODE REVIEW

**Comment by miketierney** [comment]
The code change looks good and makes sense; I'll have to check it out when I'm off work. Thanks  for submitting the PR!
